src/utils.js

```diff
@@ -1,13 +1,33 @@
 'use strict'
 
 const tempdir = require('ipfs-utils/src/temp-dir')
-const TextEncoder = require('ipfs-utils/src/text-encoder')
-const TextDecoder = require('ipfs-utils/src/text-decoder')
+const _TextEncoder = require('ipfs-utils/src/text-encoder')
+const _TextDecoder = require('ipfs-utils/src/text-decoder')
```

Comment:
Why `_` prefix here ? If there is a reason it would be a good to have a comment explaining it.

Reply:
because those aren't typed, and the name get duplicated